src/lib/structures/schemas/defaultGuildSchema.js

```diff
@@ -64,23 +49,62 @@ module.exports = KlasaClient.defaultGuildSchema
             .add("PROFANITY", "any", { default: { enabled: false, threshold: 0.99 } }))
         .add("invites", "boolean", { default: false }))
 
-    // Self Roles
-    .add("selfroles", folder => folder
-        .add("roles", "role", { array: true })
-        .add("enabled", "boolean", { default: true }))
-
-    // Level Roles
-    .add("levelroles", folder => folder
-        .add("roles", "any", { array: true, configurable: false })
-        .add("enabled", "boolean", { default: true }))
-
     // Patreon
     .add("patreon", folder => folder
         .add("enabled", "boolean", { default: false, configurable: false })
         .add("patron", "user", { configurable: false }))
 
+// New Schema Start -------
+
     // Misc
-    .add("musicVolume", "integer", { default: 100, max: 100, min: 0 })
-    .add("djOnly", "boolean", { default: false })
-    .add("levelup", "boolean", { default: false })
-    .add("leveltype", "string", { default: "guild", configurable: false });
+    .add("misc", folder => folder
+        .add("leveluptype", "string", { default: "guild", configurable: false })
+        .add("volume", "integer", { default: 100, max: 100, min: 0 }))
+
+    // Mod Logs
+    .add("modlogs", folder => folder
+        .add("logs", "any", { array: true, configurable: false })
+        .add("toggles", logs => logs
+            .add("warn", "boolean", { default: true })
+            .add("kick", "boolean", { default: true })
+            .add("ban", "boolean", { default: true })
+            .add("unban", "boolean", { default: true })
+            .add("softban", "boolean", { default: true })
+            .add("unmute", "boolean", { default: true })
+            .add("automod", "boolean", { default: true })
+            .add("mute", "boolean", { default: true })))
+
+    // All Channels
+    .add("channels", folder => folder
+        .add("modlogs", "textchannel")
+        .add("join", "textchannel")
+        .add("leave", "textchannel")
+        .add("logs", "textchannel"))
+
+    // Toggles
+    .add("toggles", folder => folder
```

Comment:
To allow users to disable a feature even if they don't want to get rid of the stored settings? I don't think I need to explain that to you, pretty self explanatory.

Reply:
for everything it would be 1 key. its not hard to set the channel again to enable it.

Reply:
I'd rather allow users to have the ability to enable and disable every module while preserving their settings.

Reply:
It's one key for most of them, this is how a lot of bots do this.
(even if they have a disable command for settings, they literally just reset channel because to enable it again they have to set a channel anyway)